oauth2/src/test/groovy/com/okta/spring/boot/oauth/AutoConfigConditionalTest.groovy

```diff
@@ -155,7 +150,8 @@ class AutoConfigConditionalTest implements HttpMock {
 
         // with properties it loads correctly
         webContextRunner().withPropertyValues(
-                "okta.oauth2.issuer=https://test.example.com/")
+                "okta.oauth2.issuer=https://test.example.com/",
+        "okta.oauth2.client-id=test-client-id", "okta.oauth2.client-secret=test-client-secret")
```

Comment:
The existing tets _should_ work as is, add new tests to test other permutations

Reply:
noted.